src/ia-sharing-options.js

```diff
@@ -0,0 +1,92 @@
+import { html, LitElement } from 'lit-element';
+import sharingOptionsCSS from './styles/ia-sharing-options.js';
+
+export class IASharingOptions extends LitElement {
+  static get styles() {
+    return sharingOptionsCSS;
+  }
+
+  static get properties() {
+    return {
+      creator: { type: String },
+      description: { type: String },
+      identifier: { type: String },
+      type: { type: String },
+    };
+  }
+
+  static sharingOptions = [{
```

Comment:
Yeah, I like that. After making singletons inside the component module, I was thinking I should probably move them externally, but hadn't yet looked into Petabox to see how we could deliver the options to it. I think we could just rely on the component to know how to build the URLs rather than the back end, with the exception of the optional subdomain.